.github/workflows/cargo-update.yml

```
@@ -0,0 +1,27 @@
+---
+name: Test
+
+on:
+  workflow_dispatch:
+  # Run every Monday
+  schedule:
+    - cron: '30 5 * * 1'
+
+jobs:
+  cargo-update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repo
+        uses: actions/checkout@v3
+
+      - name: Cargo update
+        run: cargo update
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          commit-message: "chore: Bump dependencies"
```

```suggestion
          commit-message: "build: Bump dependencies"
```
The conventional commit types, for the default Angular format, in use by the tool(s) that are likely to be used to enforce this standard are [HERE](https://github.com/commitizen/conventional-commit-types/blob/master/index.json) and `build` seems more appropriate than `chore`:
> Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)